app/Controllers/feedController.php

```diff
@@ -647,6 +647,126 @@ public function deleteAction() {
 		}
 	}
 
+	/**
+	 * This action force to clear cache of a feed.
```

Comment:
```suggestion
	 * This action force clears the cache of a feed.
```
(Or something like that.)